### ISSUE TITLE
A11y: Line item - Buttons must have discernible text

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5062]
Ensure buttons have discernible text. [https://dequeuniversity.com/rules/axe/4.10/button-name?application=AxeChrome|https://dequeuniversity.com/rules/axe/4.10/button-name?application=AxeChrome|smart-link] 

h3. Element Location:

{noformat}.line-item-collapse-button{noformat}

!Screen Shot 2024-10-10 at 09.26.00 AM.png|width=1100,height=688,alt="Screen Shot 2024-10-10 at 09.26.00 AM.png"!

!Screen Shot 2024-10-10 at 11.31.41 AM.png|width=1706,height=626,alt="Screen Shot 2024-10-10 at 11.31.41 AM.png"!

*P.S.* Same issue appears in all pages where custom product line item is available. Please check the task list below.

h3. To solve this problem, you need to fix at least (1) of the following:

* Element does not have inner text that is visible to screen readers
* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
* Element has no title attribute
* Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-741**



[created from CUS-741, comment 502171]
